### PR TITLE
Chapter dropdown and pages dropdown improvements

### PR DESCRIPTION
**Pull request by Mav7**
There is a known bug in there so far. When you try to switch from one chapter where the page in a .png to another where the page in a .jpg(and vise vera) it doesn't load the image for some reason. Other than that it's pretty smooth.

### CODE REVIEW

**Comment by Mav7** [comment]
@skippyPeanutButter Please, can you fix the canvas view for me? The div tags around the pages and the chapters dropdown button (respectively) screwed everything up as far as the UI is concerned. After all, what's nice functionality is the UI is stupid looking?

**Comment by Mav7** [comment]
Now each page that gets rendered is a link to the next page. So you can also click on the image to get the next page.

**Comment by skippyPeanutButter** [comment]
I'll give it shot and start tweaking it today

**Comment by Mav7** [comment]
@skippyPeanutButter Thanks alot man. I can't to see how nice it looks!

File: app/controllers/canvas_controller.rb [hunk 1]
```diff
@@ -6,15 +6,22 @@ def prev
   end
 
   def jump
-    $page = params[:page_index]
-    $page = $page.to_i
-    redirect_to :action => 'pages', series: params[:series], rpage: ComicsDirectoriesHelper.return_page(params[:series], "app/assets/images", $page)
+    if params[:page_index] != nil
+      $page = params[:page_index]
```
Comment (skippyPeanutButter) [inline app/controllers/canvas_controller.rb]: Maybe it might be better to cut it down to params[:page_index].to_i to save a line.

**Comment by skippyPeanutButter** [comment]
It's not displaying the chapters for the AOT series, am I missing anything? I pulled the images you uploaded.
screen shot 2015-12-27 at 8 04 50 pm

**Comment by Mav7** [comment]
@skippyPeanutButter Did you try rake db:migrate? I tried this out on my other computer and it works perfectly.

**Pull request merged**